### PR TITLE
Updating the ruby version to 2.2

### PR DESCRIPTION
**Pull request by sk33lz**
Updating the ruby version to 2.2 as some gem installs I was testing were failing to old version issues.

### CODE REVIEW

File: manifests/base.pp [hunk 1]
```diff
@@ -77,3 +77,26 @@
 
 include vagrant_setup
 include vagrant_vm
+
+class { 'apt':
+ always_apt_update => true,
```
Comment (tizzo) [inline manifests/base.pp]: Should be two spaces indented.

**Comment by sk33lz** [comment]
Crap, after re-running vagrant provision I ran into some errors. It looks like there is a bug with ruby 2.2 and puppet. https://tickets.puppetlabs.com/browse/PUP-3796

**Comment by sk33lz** [comment]
I'm downgrading to 2.1 to see how that works. If that fails, I will just go back to 1.9.1/1.9.3 and just ensure the ruby-dev package as well. I think that will get the dependencies that are needed to run mailcatcher.me.

**Comment by sk33lz** [comment]
After trying the Ruby2.1 release, I found that the mailcatcher tool requires some dependencies that requires Ruby2.2.2... Looks like we will have to revisit this later when we update Puppet to a version that will run Ruby2.2.x. 

    ERROR:  Error installing mailcatcher:
	activesupport requires Ruby version >= 2.2.2.

**Pull request closed without merging**